pulse_update/data/cli.py

```diff
@@ -51,8 +54,12 @@ def transform_args(args: typing.List[str]) -> typing.Dict[str, typing.Union[str,
 
 
 @click.group()
-def main() -> None:
-    pass
+@click.option("--connection", type=str, default="mongodb://localhost:21017/pulse", envvar="PULSE_MONGO_URI")
```

Comment:
Enough things are using the connection string now that I figured I'd put that as an option for all things to access from the group context

Reply:
👍 I keep trying to type this emoji and it keeps suggesting 🌮